website/docs/r/pi_placement_group.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_placement_group"
+description: |-
+  Manages a placement group in the Power Virtual Server cloud.
+---
+
+# ibm_pi_placement_group
+Create or delete a placement group.
+
+## Example usage
+The following example enables you to create a placement group with a group policy of affinity:
+
+```terraform
+resource "ibm_pi_placement_group" "testacc_placement_group" {
+	pi_placement_group_name   = "my_pg"
+	pi_placement_group_policy = "affinity"
```

Comment:
Format this example section

Reply:
OK I see the properties have different indents so I'll fix that. Thanks.